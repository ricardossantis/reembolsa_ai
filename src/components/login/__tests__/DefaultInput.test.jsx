import DefaultInput from "../default-input/index";
import React from "react";
import renderer from "react-test-renderer";

describe("Render Test", () => {
  it("renders DefaultInput", () => {
    const component = renderer.create(<DefaultInput/>).toJSON();
    expect(component).toMatchSnapshot();
  });
});

