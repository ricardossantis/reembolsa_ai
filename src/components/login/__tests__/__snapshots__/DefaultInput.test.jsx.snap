// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Render Test renders DefaultInput 1`] = `
<input
  className="ant-input sc-bdnylx losJSz"
  onBlur={[Function]}
  onChange={[Function]}
  onFocus={[Function]}
  onKeyDown={[Function]}
  type="text"
  value=""
/>
`;
